.azure/OneBranch.Publish.yml

```diff
@@ -80,6 +80,7 @@ parameters:
     - name: RPM
       destionations:
```

Comment:
Looks like there is a typo here `destionations` -> `destination`?

Reply:
I agree

Reply:
If there is, it's been there for a while. Fedora 40 has been published, so i assumed what's there works.  I didn't touch that line. Just duplicated the 40 line and tweaked it a bit to say 41.

Reply:
Also, the DEB section uses the same wording just a few lines away

Reply:
Mind fixing them?

Reply:
PR updated with the requested fix, though as before, I can't prove it works